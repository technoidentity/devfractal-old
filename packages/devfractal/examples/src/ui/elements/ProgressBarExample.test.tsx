import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { ProgressBar } from './ProgressBarExample'

test('ProgressBar Example', () => {
  const component: ReactTestRenderer = renderer.create(<ProgressBar />)
  expect(component.toJSON()).toMatchSnapshot()
})
