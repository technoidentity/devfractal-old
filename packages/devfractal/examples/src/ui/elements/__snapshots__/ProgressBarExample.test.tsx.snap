// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Default Progress bar
    </h1>
    <div
      className="box"
    >
      <progress
        className="progress"
        max="100"
        value="15"
      >
        15%
      </progress>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Colors
    </h1>
    <div
      className="box"
    >
      <progress
        className="progress is-large is-primary"
        max="100"
        value="15"
      >
        15%
      </progress>
      <progress
        className="progress is-link"
        max="100"
        value="30"
      >
        30%
      </progress>
      <progress
        className="progress is-info"
        max="100"
        value="45"
      >
        45%
      </progress>
      <progress
        className="progress is-success"
        max="100"
        value="60"
      >
        60%
      </progress>
      <progress
        className="progress is-warning"
        max="100"
        value="75"
      >
        75%
      </progress>
      <progress
        className="progress is-danger"
        max="100"
        value="90"
      >
        90%
      </progress>
      <progress
        className="progress is-light"
        max="100"
        value="30"
      >
        90%
      </progress>
      <progress
        className="progress is-dark"
        max="100"
        value="45"
      >
        90%
      </progress>
      <progress
        className="progress is-white"
        max="100"
        value="60"
      >
        90%
      </progress>
      <progress
        className="progress is-black"
        max="100"
        value="75"
      >
        90%
      </progress>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Sizes
    </h1>
    <div
      className="box"
    >
      <progress
        className="progress is-small"
        max="100"
        value="15"
      >
        15%
      </progress>
      <progress
        className="progress"
        max="100"
        value="30"
      >
        30%
      </progress>
      <progress
        className="progress is-medium"
        max="100"
        value="45"
      >
        45%
      </progress>
      <progress
        className="progress is-large"
        max="100"
        value="60"
      >
        60%
      </progress>
    </div>
  </section>,
]
`;
